docs/source/conf.py

```diff
@@ -24,9 +24,9 @@
 author = 'The tf-encrypted Contributors'
 
 # The short X.Y version
-version = '0.5.1'
+version = '0.5.2'
 # The full version, including alpha/beta/rc tags
-release = '0.5.1'
+release = '0.5.2-rc2'
```

Comment:
if this is the commit for the 0.5.2 release, I believe the rc tags need to be removed in each of these files -- hopefully @ianlivingstone can confirm here

Reply:
Yep! They need to be updated for each release. Ideally, these values should be centralized in one file or templated and derived at the time of the release so no one has to think about it.

Reply:
kept them in in case we needed a rc3 -- if none of you think so i'll prepare the final release